pmm/aws-staging-start.groovy

```diff
@@ -201,6 +206,7 @@ pipeline {
                                 ],
                                 "EbsOptimized": false,
                                 "ImageId": "ami-15e9c770",
+                                "UserData": "c3VkbyB5dW0gaW5zdGFsbCAteSBqYXZhLTEuOC4wLW9wZW5qZGsKCnN1ZG8gL3Vzci9zYmluL2FsdGVybmF0aXZlcyAtLXNldCBqYXZhIC91c3IvbGliL2p2bS9qcmUtMS44LjAtb3Blbmpkay54ODZfNjQvYmluL2phdmEKCnN1ZG8gL3Vzci9zYmluL2FsdGVybmF0aXZlcyAtLXNldCBqYXZhYyAvdXNyL2xpYi9qdm0vanJlLTEuOC4wLW9wZW5qZGsueDg2XzY0L2Jpbi9qYXZhYwoKc3VkbyB5dW0gcmVtb3ZlIGphdmEtMS43Cg==",
```

Comment:
Isn't this your personal credential?

Reply:
no this is commands to provision the instance you can echo them into `base64 -D` to decode them into the commands

Reply:
why can not you do it via ?
https://pmm.cd.percona.com/configureClouds/
label awscli  -> Init script

Reply:
this is for nodes we create on the fly for QA engineers to use it is not jenkins slaves